src/SFA.DAS.AssessorService.Application.Api.Client/Clients/StandardVersionClient.cs

```diff
@@ -23,7 +23,23 @@ public async Task<IEnumerable<StandardVersion>> GetAllStandardVersions()
                 return await RequestAndDeserialiseAsync<IEnumerable<StandardVersion>>(request, $"Could not get the list of standards");
             }
         }
-        
+
+        public async Task<IEnumerable<StandardVersion>> GetLatestStandardVersions()
+        {
+            using (var request = new HttpRequestMessage(HttpMethod.Get, $"/api/v1/standard-version/standards/latest"))
+            {
+                return await RequestAndDeserialiseAsync<IEnumerable<StandardVersion>>(request, $"Could not get the list of standards");
```

Comment:
Does this error message need to be changed to distinguish it from the All method or is that OK?

Reply:
Fixed

Reply:
Resolving on behalf of Ben